frontend/src/js/controllers/authCtrl.js

```diff
@@ -176,6 +176,14 @@
             }
         };
 
+        vm.checkStrength = function(password){
+            //TextBox left blank. 
+            var password_strength = utilities.check(password) ;
+            var pswd-strength-message = document.getElementById("pswd-strength-message");
```

Comment:
@Obtin Not this way, use camel case in JS (vvariable declaration) adnd dash format in css classes and id's. I hope you get it. Do these changes please. Also, use angular element selector to get element instead of using `document.getElementById`.